robottelo/ssh.py

```diff
@@ -22,8 +22,10 @@ class SSHCommandTimeoutError(Exception):
 
 
 def decode_to_utf8(text):  # pragma: no cover
-    """In python 3 all strings are already unicode, no need to decode"""
-    if six.PY2:
+    """Paramiko returns bytes object and we need to ensure it is utf-8 before
+    parsing
+    """
+    if isinstance(text, bytes):
```

Comment:
I would suggest using EAFP (https://docs.python.org/3/glossary.html#term-eafp), having in mind that:

```python
>>> 'str'.decode
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'str' object has no attribute 'decode'
>>> b'bytes'.decode
<built-in method decode of bytes object at 0x7faa1f2236e8>
```

Than you could use as:

```python
try:
    return text.decode('utf-8')
except AttributeError:
    return text
```